### ISSUE TITLE
Auto-thumbnails Not Working?

### ISSUE THREAD

**Issue by Konclan** [BenVlodgi/BEE2.4#320]
I've been trying to get a screenshot for my map for the thumbnail, but I can't seem to get the screenshot I just took (with F5) into the thumbnail when publishing just after, then I get this randomly 
![thumbnailconfusion](https://cloud.githubusercontent.com/assets/16567080/15090592/136875c6-13f2-11e6-9525-993d0d45b5d7.PNG)

**Comment by corrupt21** [comment]
You have to press F5 then you have to re-compile.

**Comment by Konclan** [comment]
Yeah I finally got it to work, but according to TeamSpen it's supposed to work without having to recompile

**Comment by Luke18033** [comment]
Is that your username in the steam window?

**Comment by Konclan** [comment]
Yes why?
